docs-website/markdown-link-check-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "ignorePatterns": [
+    {
+    "pattern": "^http://demo\\.datahubproject\\.io"
+    },
+    {
+    "pattern": "^http://localhost"
+    },
+    {
+    "pattern": "^http://www.famfamfam.com"
+    },
+    {
+    "pattern": "^http://www.linkedin.com"
+    },
+    {
+    "pattern": "\\.md$"
+    },
+    {
+    "pattern": "\\.pdl$"
+    },
+    {
+    "pattern": "\\.pdl#.*$"
+    },
+    {
+    "pattern":"\\.json$"
+    },
+    {
+    "pattern":"\\.txt$"
+    },
+    {
+    "pattern":"\\.js$"
+    },
+    {
+    "pattern": "\\.svg$"
```

Comment:
these links should also work properly, and shouldn't be ignored

e.g. the main image at the top of the repo's readme is broken right now

https://github.com/datahub-project/datahub/blob/master/README.md